renovate.json

```diff
@@ -5,11 +5,9 @@
   "automerge": true,
   "schedule": ["every weekend"],
   "ignoreDeps": ["nock"],
```

Comment:
Why nock is ignored?

Reply:
We discussed this already. nock was already ignored in dependabot. So I kept it here.

Reply:
Yes, I remember you wrote sth like that, but what is the reason it was ignored? I cannot find a reason.

Reply:
Yeah, Even I am not sure exactly on the reason. @ahmetoz do you have any idea on this?

Reply:
Don't know the commit is from dependabot and we can not access the old config anymore. the deps related to test not the source code so I would suggest removing that ignore from the config, so we could see the real reason when it occurs again 😉